<!DOCTYPE html>
<html lang="en" dir="ltr">

<head>
  <meta charset="utf-8">
  <title>Proteus</title>
  <!-- favicon -->
  <link rel="icon" href="../../../images/proteus-favicon.ico">
  <!-- css stylesheet -->
  <link rel="stylesheet" href="../../../css/blogpost.css">
  <!-- bootsrap stylesheet -->
  <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.5.2/css/bootstrap.min.css" integrity="sha384-JcKb8q3iqJ61gNV9KGb8thSsNjpSL0n8PARn9HuZOnIxN0hoP+VmmDGMN5t9UJ0Z" crossorigin="anonymous">
  <!-- HTML comment box -->
  <link rel="stylesheet" type="text/css" href="https://www.htmlcommentbox.com/static/skins/bootstrap/twitter-bootstrap.css?v=0">
  <!-- fontawesome -->
  <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.7.0/css/all.css" integrity="sha384-lZN37f5QGtY3VHgisS14W3ExzMWZxybE1SJSEsQp9S+oqd12jhcu+A56Ebc1zFSJ" crossorigin="anonymous">
  <!-- Google fonts -->
  <link href="https://fonts.googleapis.com/css2?family=David+Libre:wght@500&family=Merriweather:wght@700&family=Open+Sans&family=Montserrat:wght@600&display=swap" rel="stylesheet">
  <!-- Javascript and Jquery -->
  <script src="../../../components/footer.js" type="text/javascript" defer></script>

  <script src="https://code.jquery.com/jquery-3.5.1.slim.min.js" integrity="sha384-DfXdz2htPH0lsSSs5nCTpuj/zy4C+OGpamoFVy38MVBnE+IbbVYUew+OrCXaRkfj" crossorigin="anonymous"></script>
  <script src="https://cdn.jsdelivr.net/npm/popper.js@1.16.1/dist/umd/popper.min.js" integrity="sha384-9/reFTGAW83EW2RDu2S0VKaIzap3H66lZH81PoYlFhbGU+6BZp6G7niu735Sk7lN" crossorigin="anonymous"></script>
  <script src="https://stackpath.bootstrapcdn.com/bootstrap/4.5.2/js/bootstrap.min.js" integrity="sha384-B4gt1jrGC7Jh4AgTPSdUtOBvfO8shuf57BaghqFfPlYxofvL8/KUEfYiJOMMV+rV" crossorigin="anonymous"></script>
</head>

<body>
  <section id="navigation-bar">
  <div class="container-fluid" id="container-position">
    <nav class="navbar fixed-top navbar-expand-lg navbar-light bg-light nav-opacity">
      <a class="navbar-brand" id="website-title" href="https://www.instagram.com/proteus.iisertvm/?hl=en" target="_blank"><img class="logo" src="../../../images/proteus-logo.png" alt="logo">Proteus</a>
      <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarSupportedContent" aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
        <span class="navbar-toggler-icon"></span>
      </button>
      <div class="collapse navbar-collapse" id="navbarSupportedContent">
        <ul class="navbar-nav ml-auto">
          <li class="nav-item nav-icon">
            <a class="nav-link" href="../../../index.html">
              <strong>Home</strong>
            </a>
          </li>
          <li class="nav-item nav-icon">
            <a class="nav-link" href="../../../blog-site/blog.html">
              <strong>Series</strong>
            </a>
          </li>
          <li class="nav-item nav-icon">
            <a class="nav-link" href="../../../contact/contact.html">
              <strong>Contact Us</strong>
            </a>
          </li>
        </ul>
      </div>
  </div>
</section>

  <section id="title">
    <div class="row">
      <h1 class="heading">eDNA Metabarcoding</h1>
    </div>
    <div class="row">
      <p class="sub-head">January 15, 2022 | 4 min read</p>
    </div>
    <div class="row">
      <img src="images/A4.png" alt="cover-image" class="cover-img">
    </div>
  </section>

  <section id="blog-post">
    <div class="row">
      <div class="col-lg-8">
        <p class="article">
          Environmental DNA, as the name suggests, is a collective term that refers to nuclear/mitochondrial genome fragments of organisms collected from the environment. This could include an entire microorganism, spores, gametes, faecal matter, feathers, scales, or even bodily secretions. Traditional methods for assessing biodiversity in a community are tedious and often impractical. Hence, the novel methods ‘eDNA barcoding and metabarcoding’ have been developed.
          <br><br>
          eDNA barcoding can be broadly segregated into 6 steps:
          <br>
          <ol>
            <li>Field sampling: collection of a sample of soil, water, etc from the environment.
            <li>Precipitation or filtration (hinging on variables such as pore size and filter material) of DNA fragments from the abiotic environment.
            <li>Extraction of these DNA fragments.
            <li>Screening of the entire community.
            <li>Amplification by Polymerase Chain Reaction (carried out for the entire community using existing markers that are similar to the ones used in general sequencing).
            <li>Next-Generation Sequencing.<br><br>       The above steps differ from DNA barcoding in the following ways:

          </ol>
          <ul>
          <li>DNA barcoding, as opposed to metabarcoding, relies on a specie   s-specific approach.
          <li>The DNA of the target organism is first isolated from the sample.
          <li>PCR, both conventional and quantitative can be employed, however, quantitative PCR or qPCR is preferred for barcoding as the risk of false positives due to cross-amplification is eliminated. Furthermore, in qPCR, taxon-specific primers ensure amplification of that region of DNA that is unique to the target organism.  Probes can assist this process by securing quantitative information.
          <li>Traditional sequencing is carried out.
          <li>Databases like GenBank can be used to compare the resultant sequence to that of identified species. Barcode of Life Data Systems (BOLD) is another such sequence depot, from which gathered data and photographs can be searched for, retrieved, and utilized.
          </ul>
          <br><br>
          Although eDNA disperses and becomes diluted in aquatic environments, it can still be useful for about a fortnight, depending on exposure to UV radiations, changes in temperature, salinity, water currents, substrate availability among other factors.
          <br><br>
          Barcoding was initially used to survey microbial diversity, but more recently, has also been found useful in macroorganismal assessment. Such a process is highly advantageous in situations wherein habitats are inaccessible or the population of the target species is low. In the case of many ephemeral species, it is perhaps better to rely on eDNA from their habitat, since searching and catching such organisms for analyses is both expensive and time-consuming.
          <br><br>
          But what are these barcodes?
          <br>
          Not unlike the kenspeckle sticker on a purchased product that resembles an uneven zebra crossing, DNA barcodes are also specific sequences. The sequence, like any product, allows taxonomic identification when compared to a reliable public database such as the International Barcode of Life (iBOL) Consortium’s Barcode Library (Barcode of Life Data Systems). Every barcode has a ‘barcode gap’, a quantification of intra- and inter-specific variability. When calculated for percentage genetic distance, it represents the upper and lower limits of the aforementioned ‘barcode gap’. Common DNA barcodes include:
          <br>
          <ul>
          <li>rbcL (chloroplast ribulose biphosphate carboxylase gene), matK plastid regions; ITS i.e. internal transcribed spacer region of DNA of nuclear ribosomes—for plants
          <li>CO1 or mitochondrial cytochrome c oxidase subunit I gene (COI)—for animals
          <li>ITS in the nuclear ribosomal cistron—for fungi.
          </ul>
          <br>
          Barcoding and metabarcoding outweigh the more conventional practices on account of their non-invasive nature and high sensitivity. These novel methods serve a cornucopia of applications in pest management, monitoring native species at various stages of life, detecting invasive alien species and elusive species, to name a few.
        </p>

        <div class="class">
          <img src="images/A1.png" alt="picture">
          <p class="article">
            <em><strong>Figure 1.</strong>  <a href="https://www.google.com/imgres?imgurl=https%3A%2F%2Fmedia.springernature.com%2Foriginal%2Fspringer-static%2Fimage%2Fchp%253A10.1007%252F978-3-030-15604-6_14%2FMediaObjects%2F476359_1_En_14_Fig2_HTML.png&imgrefurl=https%3A%2F%2Flink.springer.com%2Fchapter%2F10.1007%2F978-3-030-15604-6_14&tbnid=LiTUKNDBTXBmvM&vet=12ahUKEwjGvsbR8eLwAhUbE7cAHdz8C4sQMygBegUIARCmAQ..i&docid=a5OAsx1I9PkDkM&w=713&h=738&q=edna%20metabarcoding&ved=2ahUKEwjGvsbR8eLwAhUbE7cAHdz8C4sQMygBegUIARCmAQ"> Source</a></em>
          </p>
        </div>

      </div>

      <div class="col-lg-4">
        <div class="row">
          <div class="card bg-light mb-3 ml-5" style="width: 18rem;">
            <h5 class="card-header">About This Author</h5>
            <div class="card-body">
              <p>Riya Shende is a Batch 20 BS-MS student at IISER TVM</p>
              <a target="_blank" href="https://www.instagram.com/riya_shende_"><i class="fab fa-instagram icon"></i></a>
              <a target="_blank" href="mailto:riyapshende20@iisertvm.ac.in"><i class="far fa-envelope icon"></i></a>
            </div>
          </div>
        </div>
        <div class="row">
          <div class="card bg-light mb-3 ml-5" style="width: 18rem;">
            <h5 class="card-header">Sources And Links</h5>
            <div class="card-body">
              <ol>
                <u>Article Sources:</u>
                <li><a class="card-link" href="https://www.researchgate.net/publication/273792690_" target="_blank"> How will the ‘molecular revolution’ contribute to biological recording?</a></li><br>
                <li><a class="card-link" href="https://ibol.org/barcodebulletin/" target="_blank">ARCTIC BIOSCAN: TRACKING BIODIVERSITY IN CANADA’S MIDDLE ARCTIC USING DNA </a></li><br>
                <li><a class="card-link" href="https://pubs.usgs.gov/fs/2012/3146/" target="_blank">Application of Environmental DNA for Inventory and Monitoring of Aquatic Species </a></li><br>
                <li><a class="card-link" href="https://www.frontiersin.org/articles/10.3389/fmars.2019.00674/full#B38" target="_blank">Application of Environmental DNA Metabarcoding to Spatiotemporal Finfish Community Assessment in a Temperate Embayment </a></li><br>
                <a href="https://www.instagram.com/p/CYJrZw3vx4W/" target="_blank"><i class="fab fa-instagram icon"></i></a>
                <br>
            </div>
          </div>
        </div>
      </div>
    </div>
  </section>

  <section id="more-posts">
    <div class="row">
      <div class="col-md-6">
        <a href="../../Anekdota/Anekdota_Telomerase/blog.html" target="_blank"><img src="../../Z_Preview/Telomerase.png" alt="posts"></a>
        <h3>the story behind telomerase. no caps!</h3>
        <p>May 16, 2021 | 3 min read </p>
      </div>
      <div class="col-md-6">
        <a href="../../Anthropoid/Anthropoid_Jealous/blog.html" target="_blank"><img src="../../Z_Preview/Jealous.png" alt="posts"></a>
        <h3>Do Animals Fall Prey to the Green-Eyed Monster?</h3>
        <p>May 16, 2021 | 4 min read </p>
      </div>
    </div>
  </section>

  <footer-component></footer-component>

  <!-- <section id="footer">
    <div class="row">
      <div class="col-lg-3 col-md-12">
        <h2 style="color: #fff; font-size: 2.2rem;">Find us on</h2>
        <a target="_blank" href="https://www.youtube.com/channel/UCl2xFz0fNma-cW679jgjE0A"><i class="fab fa-youtube icon"></i></a>
        <a target="_blank" href="https://www.instagram.com/proteus.iisertvm/?hl=en"><i class="fab fa-instagram icon"></i></a>
        <a target="_blank" href="https://twitter.com/proteusiisertvm"><i class="fab fa-twitter icon"></i></a>
        <a target="_blank" href="https://www.facebook.com/proteus.iisertvm"><i class="fab fa-facebook-square icon"></i></a>
      </div>
      <div class="col-lg-3">
        <h3 class="footer-head">Share the page</h3>
        <a target="_blank" class="terms" href="http://www.facebook.com/sharer.php?u=https://proteusiisertvm.github.io/Proteus/" target="_blank">Facebook</a><br>
        <a target="_blank" class="terms" href="http://www.linkedin.com/shareArticle?mini=true&amp;url=https://proteusiisertvm.github.io/Proteus/" target="_blank">LinkedIn</a><br>
        <a target="_blank" class="terms" href="https://twitter.com/share?url=https://proteusiisertvm.github.io/Proteus/&amp;text=Proteus&amp;hashtags=proteusiisertvm" target="_blank">Twitter</a>
      </div>
      <div class="col-lg-3 col-md-12">
        <h3 class="footer-head">Blogs</h3>
        <a target="_blank" class="terms" href="../../Anekdota/Anekdota_Human Soul/blog.html">Weight of The Human Soul</a><br>
        <a target="_blank" class="terms" href="../../Anthropoid/Anthropoid_Personality/blog.html">Do Animals Have Personalities</a><br>
        <a target="_blank" class="terms" href="../../Erevna/Erevna_Mammoth/blog12.html">Gene Time Bomb Hurt Mammoths</a><br>
        <a target="_blank" class="terms" href="../../../archives/archives.html">Archives</a><br>
      </div>
      <div class="col-lg-3 col-md-12">
        <h3 class="footer-head">Quick links</h3>
        <a target="_blank" class="terms" href="http://www.iisertvm.ac.in/">IISER Thiruvananthapuram</a><br>
        <a target="_blank" class="terms" href="http://biology.iisertvm.ac.in/">School of Biology</a><br>
        <a target="_blank" class="terms" href="Acknowledgements/page.html">Acknowledgements</a><br>
      </div>
      <hr style="border-top: 1px solid #fff; width: 97%; margin-top: 50px">
    </div>
    <div class="row">
      <div class="col-lg-8" style="margin: 30px auto; display: inline-block;">
        <h2 style="font-family: 'David-Libre'; font-size: 2rem; color: #fff;">Proteus</h2>
        <p style="color: grey;">Made with ❤ and a bit of code</p>
      </div>
      <div class=" col-lg-4">
        <a target="_blank" href="https://www.iisertvm.ac.in/"><img class="iiser_logo" src="../../../images/iiser_logo2.png" alt="IISER Thiruvananthapuram logo"></a>
      </div>
    </div>
  </section> -->

  <!-- Javascript cdn -->
  <script src="https://cdn.jsdelivr.net/npm/darkmode-js@1.5.7/lib/darkmode-js.min.js"></script>
  <!--HAS TO BE ABOVE THE OTHER JS SCRIPT-->

  <script src="../../../js/blog.js" type="text/javascript" charset="utf-8"></script>
</body>
          </ol>

</html>
